vespajlib/src/main/java/com/yahoo/slime/Inserter.java

```diff
@@ -7,7 +7,7 @@
  * deserializers where you can use it to decouple the actual value
  * decoding from the container where the value should be inserted.
  **/
-interface Inserter {
+public interface Inserter {
```

Comment:
the SlimeInserter class should also be made public for completeness

Reply:
Done